templates/base_index.html

```diff
@@ -28,9 +28,35 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.latest.short_version }} is here
     {% endblock notices_content %}
 
     <section class="p-strip is-deep is-bordered">
-      {% with gtm_event_label="ubuntu.com homepage" %}
-        {% include "shared/_insights_news_strip.html" %}
```

Comment:
Deleted the `_insights_news_strip` include because the homepage seems to be the only places really using it (`takeovers/templates` doesn't count).